src/css/jwplayer/imports/jwplayerlayout.less

```diff
@@ -61,8 +61,10 @@
 }
 
 .jw-plugin {
-    position: absolute;
-    bottom: (@mobile-touch-target * 1.5);
+    &:not(.jw-plugin-googima) {
+        position: absolute;
+        bottom: (@mobile-touch-target * 1.5);
+    }
```

Comment:
Rather than this `not` rule here for an element not in this project, add a new position or bottom rule in https://github.com/jwplayer/jwplayer-ads-googima/blob/master/src/css/styles.less